tests/test_easy.py

```diff
@@ -120,16 +120,40 @@ def test_snmp_get_unknown(sess_args):
 
 
 @pytest.mark.parametrize(
+    'sess_args', [sess_v1_args()]
+)
+def test_snmp_v1_get_with_retry_no_such(sess_args):
+    sess_args['retry_no_such'] = True
+
+    res = snmp_get(['iso', 'sysDescr.0', 'iso'], **sess_args)
+
+    assert res[0]
```

Comment:
These extra asserts for the value itself are not quite consistent with other unit tests, please consider removing